packages/wallet/src/test-utils/launchNode.ts

```diff
@@ -43,6 +65,17 @@ export type KillNodeParams = {
   };
 };
 
+const findBinPath = (binCommandName: string) => {
+  let binPath = npmWhich.sync(binCommandName);
+
+  if (!existsSync(binPath)) {
+    // The user might be using bun, which has a different structure for binaries inside node_modules
+    binPath = path.join('node_modules', '.bin', binCommandName);
+  }
+
+  return binPath;
+};
```

Comment:
We should deduplicate this `findBinPath` and use the one from `launchNode.ts.`

https://github.com/FuelLabs/fuels-ts/blob/9923e0a2fa02b0b1b8ac8e77bed0be42565f31f9/packages/wallet/src/test-utils/launchNode.ts#L68-L77